pkgs/os-specific/linux/linuxptp/makefile-no-prefix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/makefile b/makefile
+index 693e75d..1cc6ec6 100644
+--- a/makefile
++++ b/makefile
+@@ -38,7 +38,7 @@ incdefs := $(shell $(srcdir)/incdefs.sh)
+ version := $(shell $(srcdir)/version.sh $(srcdir))
+ VPATH	= $(srcdir)
+ 
+-prefix	= /usr/local
++prefix	=
```

Comment:
have you tried to upstream the change ? looks pretty bad not being able to override prefix.
Actually it should be more prefix ?= PREFIX

Reply:
Setting prefix as [makeFlag](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=linuxptp) actually works and is more elegant than the patch.

Reply:
it would be even better if you didn't have to specify it, as pkgs/stdenv/generic/setup.sh already provides a value for it.

Reply:
But the stdenv sets prefix only as flag for configure? Note that this package only comes with a makefile but no configure script. Without specifying prefix here everything ends up in $out/usr/local/[bin|share].